include/CppUTest/CppUTestConfig.h

```diff
@@ -199,6 +199,27 @@
 #define CPPUTEST_CHAR_BIT 8
 #endif
 
+/* Support for "long long" type */
+#ifdef CPPUTEST_USE_LONG_LONG
```

Comment:
Ok. Let me explore this packaging tonight (a couple of hours) if I'm still awake by then. I'll let you know. I think most of this PR is in the right direction so I'll make sure this is merged at least by the end of the week. It has been open too long already!

re memcpy. Entertained the idea and decided it causes more trouble. Especially when doing LONG_LONG_CMP(10, 10) you can't do a memcpy as the &10 will unlikely be a good thing :)